dashboard/modules/runtime_env/runtime_env_agent.py

```diff
@@ -216,7 +343,67 @@ def setup_plugins():
 
             return context
 
-        serialized_env = request.serialized_runtime_env
+        # Create runtime env with retry times. This function won't raise exceptions.
+        # Returns a tuple which contains result(bool), runtime env context(str), and
```

Comment:
Can you follow the proper Python-style docstring (https://docs.ray.io/en/master/ray-contribute/getting-involved.html#code-style)? Also, consider adding type annotation t

```python3
        async def _create_runtime_env_with_retry(
            runtime_env: RuntimeEnv, 
            serialized_runtime_env: str, 
            serialized_allocated_resource_instances: str
        ) -> Tuple[X, Y, Z]:
        """
         Blah blah
         
         Args:
         Returns:

        """
```